concent_api/middleman/decorators.py

```diff
@@ -0,0 +1,26 @@
+import traceback
+from functools import wraps
+
+
+def safe_execution(logger, crash_logger):
+    """
+    Decorator that allows safe execution of coroutine - no Exception is expected to be raised outside it.
+    :param crash_logger:
```

Comment:
See my other remarks. I think that a function that handles the futures would be a better solution than such a decorator.